[CUBRIDMAN-265] add note for changing LOB path from Absolute to RELATIVE (#583)
CUBRID/cubrid-manual: en/sql/datatype.rst

    ERROR: doc_t.content can not be an ORDER BY column

.. note::

>     *   From CUBRID 11.4, the locator, which represents lob file path, **changed from an absolute file path to relative**. The relative path starts from '**lob-base-path**' in databases.txt. With this, LOB file path management has become more flexible, and LOB file path problems have been reduced during database move, backup and restore.

Reviewer: ```suggestion
    *   From CUBRID 11.4, the locator, which represents the LOB file path, **changed from an absolute file path to a relative one**. The relative path starts from '**lob-base-path**' in databases.txt. With this change, LOB file path management has become more flexible, and LOB file path problems have been reduced during database moves, backups, and restores.
```

    *   For example, if the '**lob-base-path**' path defined in databases.txt is /home1/data1, an example of the locator for each CUBRID version is as follows.

        * **CUBRID 11.4** (**relative**): file:ces_004/doc_t.00001366272829040346_0773
        * up to CUBRID 11.3 (**absolute**): file:/home1/data1/ces_004/doc_t.00001366272829040346_0773

>     *   Due to the locator is stored as a relative path, if ‘**lob-base-path**’ in databases.txt is changed,

Reviewer: ```suggestion
    *   Due to the locator being stored as a relative path, if ‘**lob-base-path**’ in databases.txt is changed,
```

        * Previously created LOB data cannot be retrieved.

>         * To retrieve previously created LOB data, they should be moved to the changed '**lob-base-path**' directory.

Reviewer: ```suggestion
        * To retrieve previously created LOB data, it should be moved to the updated '**lob-base-path**' directory.
```

       ::

          # example:
          % cp -r /home1/data1/* /home2/data2
       ::

>     *   LOB data created in CUBRID 11.3 or lower, even they store the locator as an absolute file path, can be retrieved in CUBRID 11.4 without any changes.

Reviewer: ```suggestion
    *   LOB data created in CUBRID 11.3 or earlier, even though they store the locator as an absolute file path, can be retrieved in CUBRID 11.4 without any changes.

```

>     *   Advantages while database move and restore: Even if the database is moved to another location, there is no need to manually modify the LOB file path because the path is managed based on **lob-base-path**.

Reviewer: ```suggestion
    *   Advantages during database moves and restores: Even if the database is moved to another location, there is no need to manually modify the LOB file path because the path is managed based on **lob-base-path**.

```

>     *   Adoption of **relative LOB file path** is one of the notable improvements in CUBRID 11.4 and it considerably improves database management efficiency.

Reviewer: ```suggestion
    *   The adoption of the **relative LOB file path** is one of the notable improvements in CUBRID 11.4, and it significantly enhances database management efficiency.

```
